CRISPResso2/CRISPRessoReport.py

```diff
@@ -263,23 +272,23 @@ def make_multi_report_from_folder(crispresso2_info,names_arr,report_name,crispre
         folder_name = 'CRISPResso_on_%s' % name
         sub_folder = os.path.join(folder, folder_name)
         run_data = CRISPRessoShared.load_crispresso_info(sub_folder)
-        if not 'report_filename' in run_data:
+        if not 'report_filename' in run_data['running_info']:
```

Comment:
I think I had this check in case the run_data couldn't be loaded (e.g. contained no keys). Should we check for the presence of 'running_info' key here or in load_crispresso_info? Probably one would be good.

Reply:
Really good catch, I will have it check for `running_info` instead.